numba/tests/test_unicode.py

```diff
@@ -154,6 +158,7 @@ def setUp(self):
 UNICODE_EXAMPLES = [
     'ascii',
     '12345',
+    '-12345',
```

Comment:
The `UNICODE_EXAMPLES` are used by many tests. Whereas the strings that are prefixed with `+` and `-` are quite specific to your use-case. You might be better off using them in the zfill test only.